### DISCUSSION TITLE
选项页打开空白的解决方案

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2764] [重要通知]
## 问题描述

当打开选项发现如下界面

![image](https://user-images.githubusercontent.com/81074/131776047-1dc723ac-f0e9-4f7a-89c7-6f90f8a52b0e.png)

## 解决方案

[同步助手 1.2.0](https://github.com/Kenshin/simpread/discussions/4049#discussioncomment-5086922) 内置了数据中心功能，里面包含了 **数据结构修复功能**。

### 说明

已运行 1.2.0 版同步助手，可直接访问此地址 http://localhost:7026/database/fixed 根据提示修复即可，关于此功能的详细说明 👉 [请看这里](https://simpread.pro/docs/#/数据中心?id=修复配置文件)。

### 视频操作

https://user-images.githubusercontent.com/81074/221127248-56a5591b-c704-41f0-9f7c-be270f56ab31.mp4

## 关联

- [稍后读打开后空白的解决方案](https://github.com/Kenshin/simpread/discussions/2727)

- [选项稍后读打开后一片空白](https://github.com/Kenshin/simpread/issues/1972)

- [清除浏览器缓存后打不开稍后读](https://github.com/Kenshin/simpread/issues/2723)

**Comment by mjysci** [comment]
### 补充一下

也可能是同步助手的同步文件夹未设置。   

### 复现

新设备的浏览器装好扩展，导入最新的配置文件 `simpread_config.json`，选项页卡在图标跳动动画。  

### 原因

旧设备中扩展配置好和同步助手读取同步文件夹的配置，而新设备中同步助手的同步文件夹路径为空，于是扩展读取不到正确配置。  

### 解决

同步助手 -> 同步 -> 同步文件夹路径，设置好正确的路径，F5 刷新选项页即可。

**Reply by Kenshin** [reply]
谢谢提示 👍

不过，这种情况很少见~ 😂 教程中这个错误一般是由于配置文件损坏造成的，跟你遇到是两种问题 😏 

多设备的同步也可以看下 https://www.yuque.com/kenshin/simpread/dgmm5q

**Reply by mjysci** [reply]
啊哈，我在 [文档中心](http://ksria.com/simpread/docs/#/) 翻了下没找到相关内容，于是想当然先导入配置然后碰到这个现象。  
应该去 [教程中心](https://www.yuque.com/kenshin/simpread/) 看看的，如上所述，应该是先配置同步助手再打开扩展导入配置。 
谢谢了！

**Reply by Kenshin** [reply]
小事情 😄
